### PR TITLE
my first commit

### PR DESCRIPTION
**Pull request by IgorKow**

### CODE REVIEW

**Comment by WildTiger404** [comment]
create git ignore and delete obj and bin folders

**Comment by WildTiger404** [comment]
where is the cycle to draw all figures?

File: Hexagon.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+
+namespace Lab_1_1
+{
+    class Hexagon: Figure
+    {
+        int x1, y1, x2, y2, x3, y3, x4, y4, x5, y5, x6, y6;
```
Comment (WildTiger404) [inline Hexagon.cs]: if you use Points it's more accurate to store Point collection than this huge amount of coordinates that you don't need in future

Comment (WildTiger404) [reply Hexagon.cs]: You can also take out this collection into Figure class

File: Hexagon.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+        public Hexagon(int x1, int y1, int x2, int y2, int x3, int y3, int x4, int y4, int x5, int y5, int x6, int y6)
```
Comment (WildTiger404) [inline Hexagon.cs]: Also if you work with Points, you should use points in constructor too

**Review by WildTiger404 — CHANGES_REQUESTED** [review]

File: Hexagon.cs (hunk 2 above)
Comment (IgorKow) [reply Hexagon.cs]: I didn't understand the comments about the constructor. Do you mean the need to use a collection of points as an input parameter?

Comment (WildTiger404) [reply Hexagon.cs]: yep, yo can do this

**Review by WildTiger404 — APPROVED** [review]

**Pull request closed without merging**